main.py

```diff
@@ -540,9 +569,11 @@ def perplexity_eval(model, testenc, args, dev):
     device = "cuda" if torch.cuda.is_available() else "cpu"
 
     print("============  Loading model... ============")
-    model = get_model(args.model_path, args.dtype).train(False)
+    model = get_model(args.model_path, args.load_quantization_pt, args.load_quantization, args.dtype).train(False)
```

Comment:
Do not pack all functions into get_model(). Have an IF statement instead:
```
if load_quantized in args:
    model = load_model(...)
else:
    model = get_model(...)
    
```

Reply:
Why not ?

Reply:
Although this may be good approach moving forward, but currently I would not separate the function into two